api/src/controllers/communityUserController.js

```diff
@@ -101,6 +101,54 @@ const getAllMembers = async (req, res) => {
   }
 }
 
+// @desc Get the community-users
+// @route GET /api/community-users/community/:id/details
+// @access Public
+
+const getAllMemberDetails = async (req, res) => {
+  try {
+    const data = await db.CommunityUser.findAll(
+      {
+        where: { communityId: req.params.id, active: true },
+        attributes: ['id', 'userId', 'role'],
+        include: [{
+          model: db.User,
+          attributes: ['firstName', 'lastName', 'email', 'phone', 'dateOfBirth']
+        }],
+        required: true
+      }
+    )
+
+    // flattening the array to show only one object
+    const newArray = data.map(item => {
+      const { userId, role, id } = item.dataValues
+      const { ...rest } = item.user
```

Comment:
this line is not required can be used directly below in place of rest.datavalues